website/content/docs/k8s/installation/vault/data-integration/bootstrap-token.mdx

```diff
@@ -22,8 +18,12 @@ To use an ACL bootstrap token stored in Vault, we will follow the steps outlined
 
 ### Setup per Consul datacenter
   1. Create Vault Kubernetes auth roles that link the policy to each Consul on Kubernetes service account that requires access.
-  1. Configure the Vault Kubernetes auth role in the Consul on Kubernetes helm chart.
+  1. Update the Consul on Kubernetes helm chart.
 
+## Prerequisites
+Prior to setting up the data integration between Vault and Consul on Kubernetes, you will need to have:
+1. Read and completed the steps in the [Systems Integration](/docs/k8s/installation/vault/systems-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
+2. Read the [Data Integration Overview](/docs/k8s/installation/vault/data-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
 ## One time setup in Vault
 ### Generate and Store the Secret in Vault
```

Comment:
```suggestion
## Store the secret in Vault

You only need to complete the following procedure once.

1. First, issue the following command to generate and store the ACL bootstrap token in Vault:
    ```shell-session
    $ vault kv put secret/consul/bootstrap-token token="$(uuidgen | tr '[:upper:]' '[:lower:]')"
    ```
1. Next, create a Vault policy that authorizes the desired level of access to the secret:
```
I didn't finish this building this out, but the idea is to use the bullets from the overview as section headings and put the current subheadings into a steps as to group related actions. The result should be:

Title (h1)
What this is about

Overview (h2)
General procedure

Prerequisites (h2)
Stuff user needs to do ahead of time

Store the secret in Vault (h2)
Steps to create and store secret

Create Vault policy (h2)
Steps to create and link the policy for enabling access

Create Vault Authorization Roles in Consul (h2)
Steps to complete this action

Update Consul on Kubernetes helm chart (h2)
Steps to complete this action


This should be the template for similar content in this PR.